apps/protailwind/src/video/exercise-overlay.tsx

```diff
@@ -391,14 +383,16 @@ const BlockedOverlay = () => {
         <>
           <div className="z-20 flex h-full w-full flex-shrink-0 flex-col items-center justify-center gap-10 p-5 pb-10 text-center text-lg leading-relaxed sm:gap-5 sm:p-10 sm:pb-16 xl:flex-row">
             <div className="flex w-full max-w-xl flex-col items-center justify-center gap-2">
-              <div className="relative flex items-center justify-center rounded-full bg-white p-5">
-                <Image
-                  src={module.image}
-                  width={110}
-                  height={110}
-                  alt={module.title}
-                />
-              </div>
+              {module.image && (
+                <div className="relative flex items-center justify-center rounded-full bg-white p-5">
+                  <Image
+                    src={module.image}
+                    width={110}
+                    height={110}
+                    alt={module.title}
+                  />
+                </div>
+              )}
```

Comment:
this has bitten us a few times! The `SanityDocument` type is only marginally better than `any`